portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/messageboards/MessageboardsUsecase.testcase

```diff
@@ -36,7 +36,6 @@ definition {
 	@refactorneeded
 	test AddMBThreadWithLinkToDocument {
 		property portal.acceptance = "true";
-		property testray.component.names = "WYSIWYG";
```

Comment:
Hi @austinchiang , can we include this testray component back? I use it to filter for tests that would be good for changes from the frontend-editor module. Otherwise I'd be making duplicate test or more complicated test.property filter.

Reply:
Sure I'll add it back

Reply:
See https://github.com/liferay-lima/liferay-portal/pull/1731